doc/requirements.txt

```
@@ -1,4 +1,7 @@
-numpydoc
-ipython
-nbsphinx
-sphinx_rtd_theme
+python=3.8.5
+numpydoc=1.1.0
+ipython=7.17.0
+pygments=2.4.1
```

I just fixed this same issue in a different repo, and I used 
```suggestion
Pygments>=2.6
```